### DISCUSSION TITLE
moba的SFTP如何获得root权限

### DISCUSSION THREAD

**Discussion by kettly1260** [RipplePiam/MobaXterm-Chinese-Simplified#74] [Q&A]
以前是有个🔑的选项，现在没了，大家知道这个设置在哪吗

**Comment by Y-ASLant** [comment]
![PixPin_2024-12-09_16-43-19](https://github.com/user-attachments/assets/2b861e1d-e155-439e-b044-c7be84640bac)

**Reply by Y-ASLant** [reply]
这个嘛？

**Reply by kettly1260** [reply]
不是，以前左边的文件管理选项可以直接编辑特权用户文件夹，现在不行了
![image](https://github.com/user-attachments/assets/c12c74b7-8c61-4608-8d79-643a64fc8f45)

**Comment by RipplePiam** [comment]
侧边栏拉宽一点，或者右键
![image](https://github.com/user-attachments/assets/da1c7d69-7341-4dfe-a2b1-1db6665dd61e)

**Reply by kettly1260** [reply]
不是这个，以前比如说我用user账号访问/etc/内的文件，可以直接双击，利用自带的文件管理器编辑，点击上面🔑选项就可以直接保存，也可以直接删除/etc/内的文件，现在就会提示权限不足

**Reply by RipplePiam** [reply]
你可能记错了，没有权限的用户不可能直接编辑。
不知道你说的黄色小钥匙指的是什么，你这个图标要么就是设置里的密码管理器。
抛开这个工具，”可以直接删除/etc/内的文件，现在就会提示权限不足“ 可以直接删除是不现实的

**Reply by kettly1260** [reply]
绝对不可能记错，我以前一直就用这个功能编辑nginx的conf文件，就是记不起来是哪个版本了

**Reply by RipplePiam** [reply]
> 绝对不可能记错，我以前一直就用这个功能编辑nginx的conf文件，就是记不起来是哪个版本了

我知道你说的 MobaXterm 有这个功能，你说的文本编辑器是 MobaTextEditor，你去打开一个当前登录用户有权操作的文件编辑一下就知道了；
我意思是你的环境可能和以前不一样了，比如以前你登录用户是有权操作的，现在因为什么原因没有这个权限了；
